hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntry.java

```diff
@@ -36,9 +36,16 @@
 import com.google.common.annotations.VisibleForTesting;
 
 /**
- * Helper class used inside {@link BlockOutputStream}.
+ * A BlockOutputStreamEntry manages the data writes into the DataNodes.
+ * It wraps BlockOutputStreams that are connecting to the DataNodes,
+ * and in the meantime accounts the length of data successfully written.
+ *
+ * The base implementation is handling Ratis-3 writes, with a single stream,
+ * but there can be other implementations that are using a different way, like
+ * EC writes where an entry manages the write into a block group via multiple
```

Comment:
Let's avoid to have EC related docs in master code.

Reply:
Done, thank you for noticing it.